content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -233,6 +233,115 @@ server {
 }
 {{< /text >}}
 
+## 404 or 503 error while accessing headless services
+
+Assume Istio is installed with the following configuration:
+
+- `mTLS mode` set to `STRICT` within the mesh
+- `meshConfig.outboundTrafficPolicy.mode` set to `ALLOW_ANY`
+
+Consider `nginx` is deployed as a `StatefulSet` in the default namespace and a corresponding `Headless Service` is defined as shown below:
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+spec:
+  ports:
+  - port: 80
+    name: web      # Port name
+  clusterIP: None  # Creates a Headless Service
+  selector:
+    app: nginx
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: web
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  serviceName: "nginx"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: k8s.gcr.io/nginx-slim:0.8
+        ports:
+        - containerPort: 80
+          name: web
+{{< /text >}}
+
+Let us assume we have a [sleep]({{< github_tree >}}/samples/sleep) pod `Deployment` as well in the default namespace.
+
+Now, let us consider 3 possible scenarios based on the port `name` defined in the headless service:
+
+1. Port `name` set to `web` (as shown in the example above) or any other custom name
+
+    - Request without Host header:
```

Comment:
nit: "without Host header" is not possible in HTTP/1.1. There is a host header... its just set to the IP (`10.1.1.171`) in this case.

I know what you mean hear, but its not technically 100% accurate

Reply:
Yes, the host header was set to 10.1.1.171. So, once istio/istio#34680 is merged in, _curl <POD_IP>_ will work straight away when port name is set to "web" for example (basically auto http headless will work). I'll remove this section then?